package.json

```diff
@@ -27,10 +27,13 @@
   },
   "devDependencies": {
     "acquit": "1.x",
-    "istanbul": "0.3.5",
+    "nyc": "15.1.x",
     "mocha": "5.x"
   },
   "engines": {
     "node": ">=6.0.0"
-  }
+  },
+  "files": [
```

Comment:
I'm generally not a big fan of using 'files', because I think it introduces the risk of forgetting to add a new file to `files` and shipping a broken build. What's the benefit of using files? Is it just for unpkg?

Reply:
like i already wrote in mpath's PR:

> Is files just for unpkg?

no, unpkg is to show the selected published "unzipped" version (like manually downloading a version from npm and then "unzipping" it and inspecting it), "files" (or like in mongoose "directories") is used to tell npm / yarn to only package these files into the archive and only publish those files (some files like "LICENSE" and "package.json" are always included)

> I generally prefer to not ship with files, because adding a file but forgetting to add it to files is an easy mistake to make.

this is true, but if the project structure is kept (as in everything in `src` / `lib` / `build` / `dist`), then there should never be a non-working package once properly defined, though if adding files (or renaming) like a changelog, then it could be forgotten

in the case of acquit-ignore i had decided to just go with `files` over `directories`, because of the root `index.js`, which (from what i know) could not be easily included in `directories` only would still require `files` for just that

Reply:
In that case, let's remove `files` for now. Other than that, this looks good.

Reply:
> In that case, let's remove files for now. Other than that, this looks good.

i originally had started this PR with the intention to add the `files` property, because otherwise it would ship with unnecessary files (like tests)

Reply:
Can we use `.npmignore` instead? I'd rather explicitly block unnecessary files from shipping, rather than having to explicitly list out files.

Reply:
> Can we use .npmignore instead? I'd rather explicitly block unnecessary files from shipping, rather than having to explicitly list out files.

kinda forgot this file is supported, yes i will change it

Reply:
changed in 9def69af6b14ff5e140e7d46a6f9be813eba2a1a